react/src/DatePicker/index.ts

```diff
@@ -1 +1,2 @@
 export { DateInput as default } from './DateInput'
+export { DateRangeInput } from './DateRangeInput'
```

Comment:
Do we not need to export both the singular date picker and the range picker?

Reply:
Yeah, trying to debug issued with the datepicker, had a runtime error earlier when importing to phas